From 1f6c4251d23ba6117943440bc1e3f43fe691da3a Mon Sep 17 00:00:00 2001
From: Dan Callaghan <dcallagh@redhat.com>
Date: Sun, 27 Sep 2015 20:21:49 +1000
Subject: [PATCH 4/4] httpdomain: bump domain data version

Upstream should have bumped this to 1 (or something greater than 0) for
the 1.4.0 release since the expected contents of the data dict changed.
Using '0.1' here in Fedora with the expectation that it will be bumped
properly in a future upstream release.
---
 httpdomain/sphinxcontrib/httpdomain.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/httpdomain/sphinxcontrib/httpdomain.py b/httpdomain/sphinxcontrib/httpdomain.py
index faa50f2..27f5871 100644
--- a/httpdomain/sphinxcontrib/httpdomain.py
+++ b/httpdomain/sphinxcontrib/httpdomain.py
@@ -628,6 +628,7 @@ class HTTPDomain(Domain):
         'copy': {},
         'any': {}
     }
+    data_version = '0.1'
 
     indices = [HTTPIndex]
 
-- 
2.9.4

